src/cfnlint/custom_rules.py

```diff
@@ -3,17 +3,18 @@
 SPDX-License-Identifier: MIT-0
 """
 # pylint: disable=W0108
-# pylint: disable=W0622
+
 import logging
 import cfnlint
+import cfnlint.runner
 import cfnlint.customRules.Operators
 import cfnlint.customRules.Rule
 
 LOGGER = logging.getLogger(__name__)
-Operator = {'EQUALS': lambda x, y: cfnlint.customRules.Operators.equalsOp(x, y),
+Operator = {'EQUALS': lambda x, y, z: cfnlint.customRules.Operators.equalsOp(x, y, z),
             'PLACEHOLDER': lambda x, y: LOGGER.debug('Placeholder Op')}
```

Comment:
Won't all operators receive the same arguments? `Operator[rule.operator](template, rule, resource_properties)`

Reply:
Yes! They will all get the exact same arguments. I'll update the placeholder again for correctness, but this is subject to change in the near future due to some of the project requirements.

Specifically, custom error messages and/or error levels would require some adjustments there.

Reply:
Changed in new patch